sqlc.yaml

```diff
@@ -137,6 +137,143 @@ sql:
       - sqlc/db-prepare
       # - postgresql-query-too-costly
       - postgresql-no-seq-scan
+  - engine: "postgresql"
+    queries: "common/configuration/sql/queries.sql"
+    schema: "backend/controller/sql/schema"
+    database:
+      uri: postgres://localhost:15432/ftl?sslmode=disable&user=postgres&password=secret
+    gen:
+      go:
+        package: "sql"
+        sql_package: "pgx/v5"
+        out: "common/configuration/sql"
+        emit_interface: true
+        query_parameter_limit: 3
+        overrides:
```

Comment:
I mentioned this before, but can you deduplicate this with [YAML anchors](https://support.atlassian.com/bitbucket-cloud/docs/yaml-anchors/)?

Reply:
Oof thanks for the reminder! fixed